flake.nix

```diff
@@ -1,6 +1,6 @@
 {
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-22.11";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
```

Comment:
Any specific reason for this change ?

Reply:
Yes, to get access to Bazel version 6.3.2, which is not yet available in NixOS-22.11.